tests/marqo_test.py

```diff
@@ -152,6 +153,8 @@ def setUpClass(cls) -> None:
         cls.generic_test_index_name = 'test-index'  # used as a prefix when index is created with settings
         cls.generic_test_index_name_2 = cls.generic_test_index_name + '-2'
 
+        cls.indexes_to_cleanup = set()
```

Comment:
@pandu-k that's exactly what we want, purpose of this field is to reduce the amount of calls to delete_documents api. so with making it that way will make it so that class cleans up only those indices that were used during it's own tests.